assets/dev/js/frontend/utils/swiper.js

```diff
@@ -17,7 +17,8 @@ export default class Swiper {
 		container.closest( '.elementor-widget' )?.classList.add( 'e-widget-swiper' );
 
 		return new Promise( ( resolve ) => {
-			elementorFrontend.utils.assetsLoader.load( 'script', 'swiper' )
+			elementorFrontend.utils.assetsLoader.load( 'style', 'swiper' )
```

Comment:
Load Swiper styling on the initiation of a new Swiper JS Class. This class is used in all Swiper widgets that are using the `elementorFrontend.utils.swiper` to instantiate a new Swiper.